src/main/java/seedu/address/model/delivery/exceptions/DeliveryNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.delivery.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified delivery.
+ */
+public class DeliveryNotFoundException extends RuntimeException {}
```

Comment:
Good question, I was just following the AB3 logic. Maybe it is because the system cannot handle these wrong inputs to we should use RunTimeException to catch these errors? Ask forum? lol